index.html

```diff
@@ -0,0 +1,131 @@
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc" class="flex flex-row">
+      
+      <div class="basis-1/4">
+        <nav
+          id="navbar"
+          class="flex flex-col lg:fixed h-screen lg:w-1/4 z-10 sm:max-h-23 sm:absolute sm:w-full min-w-290 top-0 lg:overflow-hidden bg-gray-400 space-x-2"
+        >
+          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
+            Our Technical Page
+          </header>
+          <div
+            class="flex flex-col justify-evenly space-x-2 overflow-y-auto overflow-x-hidden"
+          >
+            <a href="#our_project" class="nav-link uppercase">our project</a>
+            <a href="#html" class="nav-link uppercase">html</a>
+            <a href="#css" class="nav-link uppercase">css</a>
+            <a href="#js" class="nav-link uppercase">js</a>
+            <a href="#php" class="nav-link uppercase">php</a>
+          </div>
+        </nav>
+      </div>
+
+      <div class="basis-3/4">
+        <section class="main-section" id="our_project">
+          <header class="uppercase font-bold">our project</header>
+          <p>This is the project we have done</p>
+          <p>We have done best</p>
+        </section>
+        <section class="main-section" id="html">
+          <header class="uppercase font-bold">html</header>
+          <p>
+            HTML is the standard markup language for Web pages. With HTML you
+            can create your own Website. HTML is easy to learn - You will enjoy
+            it!
+          </p>
+          <p>Learning with HTML "Try it Yourself"</p>
+        </section>
+        <section class="main-section" id="css">
+          <header class="uppercase font-bold">css</header>
+
+          <p>
+            CSS is the language we use to style a Web page. What is CSS? CSS
+            stands for Cascading Style Sheets CSS describes how HTML elements
+            are to be displayed on screen, paper, or in other media CSS saves a
+            lot of work. It can control the layout of multiple web pages all at
+            once External stylesheets are stored in CSS files
+          </p>
+          <p>This means css</p>
+
+          <ul>
+            <li>1</li>
+            <li>2</li>
+            <li>3</li>
+            <li>4</li>
+            <li>5</li>
+          </ul>
+        </section>
+        <section class="main-section" id="js">
+          <header class="uppercase font-bold">js</header>
+
+          <p>
+            JavaScript Can Change HTML Content One of many JavaScript HTML
+            methods is getElementById(). The example below "finds" an HTML
+            element (with id="demo"), and changes the element content
+            (innerHTML) to "Hello JavaScript":
+          </p>
+          <p>That is javascript</p>
+          <code> x=5; y=6; z=x+y; </code>
+          <code></code>
+          <code></code> <code></code> <code></code>
```

Comment:
Please add something here

Reply:
- [ ] In here, I would suggest providing a more descriptive suggestion like:

"For better code readability, I would suggest either removing these unused elements or filling them up with some content, in this way you can make sure your project looks cleaner and contains all the necessary information :heavy_check_mark: "

This comment provides an explanation and also a pro consequence of implementing the change :100:

Reply:
Is that okay? Do I need to add something?